cloudbuild.yaml

```diff
@@ -20,6 +20,11 @@ steps:
         '--tag=gcr.io/oak-ci/oak:latest',
         '.',
       ]
+    env:
+      [
+        "DOCKER_BUILDKIT=1",
```

Comment:
Could you also add a short comment here to explain what this is for, and / or a reference to some online page.